opsdetails.tex

```diff
@@ -25,13 +25,13 @@ \subsection{Ops Cost details}\label{sec:opsdetails}
 For Chile the rack costs are outlined in \tabref{tab:rackCostChile}.
 \input{rackCostChile}
 
+For Chile the power costs are outlined in \tabref{tab:powerCostChile}.
```

Comment:
In the summary we call this overhead chile .. means we could add something else if we needed to so perhaps mention this link.